### ISSUE TITLE
User should be able to view collaborators & easily add a collaborate when viewing a space

### ISSUE THREAD

**Issue by serenamarie125** [openshiftio/openshift.io#4330]
When viewing a space, the user should
1 - be able to view the collaborators without changing context
2 - be able to add a collaborator without changing context

**Comment by serenamarie125** [comment]
Initial proposal for internal review : https://redhat.invisionapp.com/share/7MO2VNOV5JB#/320201485_OSIO-4330-01
@muruGanesan @sdash-redhat @sunilmalagi @Veethika @parvathyvr @catrobson @AshishDurgude

**Comment by serenamarie125** [comment]
@catrobson yes - PatternFly shows something here: option 1 example for the [popover](https://www.patternfly.org/pattern-library/widgets/#popover)

**Comment by catrobson** [comment]
@serenamarie125 ok, just be aware this will need to be custom built in OSIO since we don't import Bootstrap in to the project.

**Comment by AshishDurgude** [comment]
Hi, @serenamarie125 and @catrobson  I have worked on the visual design on this issue. Here is the InVision link: https://redhat.invisionapp.com/share/MVOBQ1ZRXHZ

**Comment by serenamarie125** [comment]
@joshuawilson @christianvogt can we get this on the dev backlog? Once confirmed, I'll pull off the UX tags.

**Comment by joshuawilson** [comment]
Story created at https://openshift.io/openshiftio/Openshift_io/plan/detail/752

**Comment by andrewazores** [comment]
There are two screens in the design showing the popover component. In the second case there is a search widget added to the top of the popover. Is the intent to display the search widget only when the scroll bar also displays (ie when the list overflows)? I'm not sure if this will be practical to implement.
